merge_duplicate_contacts/wizard/merge_contact_view.xml

```diff
@@ -1002,19 +1002,32 @@
         <field name="model">base.partner.merge.automatic.wizard</field>
         <field name="inherit_id" ref="base.base_partner_merge_automatic_wizard_form" />
         <field name="arch" type="xml">
```

Comment:
Please set the `<field name="mode">primary</field>` otherwise you have an extension are and influencing the base view `base.base_partner_merge_automatic_wizard_form` which is probably not good.